files/docker-php.conf

```diff
@@ -17,5 +17,6 @@ DirectoryIndex index.php index.html
 	RewriteRule ^index\.php$ - [L]
 	RewriteCond %{REQUEST_FILENAME} !-f
 	RewriteCond %{REQUEST_FILENAME} !-d
+	RewriteCond %{REQUEST_URI} !server-status
```

Comment:
To allow apache to handle server-status route rather than forwarding to wordpress server routes.